rest/src/routes/networkRoutes.js

```diff
@@ -62,12 +68,15 @@ module.exports = {
 
 		server.get('/network/fees/transaction', (req, res, next) => {
 			const numBlocksTransactionFeeStats = services.config.numBlocksTransactionFeeStats || 1;
-			return db.latestBlocksFeeMultiplier(numBlocksTransactionFeeStats).then(feeMultipliers => {
+			const nodeProperties = readAndParseNodePropertiesFile();
```

Comment:
Property files should be cached somewhere and not lead each rest request.

Create a const on top of the route definition?

Reply:
Move `const nodeProperties = readAndParseNodePropertiesFile(); `aboce server.get(